examples/jsm/renderers/webgpu/WebGPUUniform.js

```diff
@@ -19,21 +19,35 @@ class WebGPUUniform {
 		this.value = value;
 
 	}
+	
+	getValue() {
+		
+		return this.value;
+		
+	}
 
 }
 
 class FloatUniform extends WebGPUUniform {
 
-	constructor( name, value = 0 ) {
+	constructor( nodeUniform ) {
```

Comment:
I think it's better if the classes that extend `WebGPUUniform` do not know about the node material. Hence, I prefer the previous ctor signature.